src/devices/sonoff.ts

```diff
@@ -950,13 +950,21 @@ const definitions: DefinitionWithExtend[] = [
         exposes: [e.cover_position(), e.battery()],
     },
     {
-        zigbeeModel: ['Z111PL0H-1JX', 'SA-029-1', 'SA-028-1'],
+        zigbeeModel: ['SA-029-1', 'SA-028-1'],
         model: 'SA-028/SA-029',
         vendor: 'SONOFF',
         whiteLabel: [{vendor: 'Woolley', model: 'SA-029-1'}],
         description: 'Smart Plug',
         extend: [onOff(), forcePowerSource({powerSource: 'Mains (single phase)'})],
```

Comment:
Please add it to here instead, I guess the SA-029 and SA-028 also don't support it.

Reply:
Thanks for the review. Though seems like there are different versions/firmware of the SA-028 being sold. One with power on behaviour (SA-028 modelId) and one without (Z111PL0H-1JX) with exactly the same packaging and plug type. 

Cant say for SA-029 but it is suppose to be only a different socket type but who then who knows what they decide to enable on the firmware ![image](https://github.com/user-attachments/assets/cb9febb6-7c89-4dc9-bac3-3d2e0a407751)

This is the db entry for the other SA-028 with power on behavior

```

{"id":117,"type":"Router","ieeeAddr":"0x7cb94c61881d0000","nwkAddr":22814,"manufId":4742,"manufName":"Sonoff","powerSource":"Mains (single phase)","modelId":"SA-028-1","epList":[1,242],"endpoints":{"1":{"profId":260,"epId":1,"devId":9,"inClusterList":[0,3,4,5,6,4096,64599,64672],"outClusterList":[25],"clusters":{"genBasic":{"attributes":{"stackVersion":0,"hwVersion":0,"zclVersion":8}},"genOnOff":{"attributes":{"onOff":0,"startUpOnOff":255}}},"binds":[{"cluster":6,"type":"endpoint","deviceIeeeAddress":"0x00124b002f8b1364","endpointID":1},{"cluster":25,"type":"endpoint","deviceIeeeAddress":"0x00124b002f8b1364","endpointID":1}],"configuredReportings":[{"cluster":6,"attrId":0,"minRepIntval":0,"maxRepIntval":65000,"repChange":1}],"meta":{}},"242":{"profId":41440,"epId":242,"devId":97,"inClusterList":[],"outClusterList":[33],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":1,"stackVersion":0,"hwVersion":0,"dateCode":"20210610-BL","swBuildId":"1.0.5","zclVersion":8,"interviewCompleted":true,"meta":{"configured":332242049},"lastSeen":1734895050344}
```

Anyhow I saw there's the S26R2ZB without power on behaviour maybe I can move it there? Instead of creating a completely new separate model for this to pass the test Wdyt?